bindings/swift/CONTRIBUTING.md

```diff
@@ -0,0 +1,77 @@
+# Contributing
+
+- [Contributing](#contributing)
+  - [Setup](#setup)
+    - [Using a dev container environment](#using-a-dev-container-environment)
+    - [Bring your own toolbox](#bring-your-own-toolbox)
+  - [Build](#build)
+  - [Test](#test)
+
+## Setup
+
+### Using a dev container environment
```

Comment:
Do you have our dev container tested? Maybe we should install swift?

Reply:
Let's create a new issue for this. I think we can merge this PR first.

Reply:
The dev container is actually unable to build the binding, it can only be used to edit code currently. Maybe we can track it later or remove this section for now?

Reply:
> Maybe we can track it later or remove this section for now?

Yes, we can track it later and remove it for now.

Reply:
PR updated.